src/Command.ts

```diff
@@ -0,0 +1,73 @@
+import { exec } from "child_process"
+import { dirname } from "path"
+
+import { commands, OutputChannel, Uri, window, workspace } from "vscode"
+
+export const COMMAND_INSTALL = "npm-outdated.install"
+export const COMMAND_NOTIFY = "npm-outdated.notify"
+
+type PackageManager = "npm" | "yarn"
+
+const getPackageManager = async (uri: Uri): Promise<PackageManager> => {
```

Comment:
Could we use VS Code's built in `npm.packageManager` setting for this?

Reply:
I belive so! I will check that right now.

Reply:
Done! I was a nice catch.

https://github.com/mskelton/vscode-npm-outdated/pull/11/commits/e3dd6e5b1867624b58d630ea176fc79ec4dfe281

I had to put `vscode.npm` as dependency of the extension, since the command belongs to this extension. Anyway, I believe it's a basic extension for this one to work, since they are totally related. Also, it's a hidden extension, so hardly anyone would disable it.